base_tier_validation/models/tier_validation.py

```diff
@@ -154,7 +154,14 @@ def _validate_tier(self, tiers=False):
             'done_by': self.env.user.id,
             'reviewed_date': fields.Datetime.now(),
         })
-        # TODO: add message_post
+        for review in user_reviews:
+            rec = self.env[review.model].browse(review.res_id)
```

Comment:
Could you put this in a hood method, something like `_notify_tier_change` or similar. This way it could be disabled/changed easily in custom modules or for specific models even if they have chatter.

Reply:
Done!